### ISSUE TITLE
The old log system had a call that would actually log a specific message only once per user session, 

### ISSUE THREAD

**Issue by christofmuc** [christofmuc/KnobKraft-orm#223]
Ah, I have a note somewhere about this. The old log system had a call that would actually log a specific message only once per user session, I think this is one of them. I'll open a new issue to get this fixed, very annoying.

_Originally posted by @christofmuc in https://github.com/christofmuc/KnobKraft-orm/issues/218#issuecomment-1432216518_

**Comment by christofmuc** [comment]
Fixed with today's 2.0.3 release.
